tools/benchmark_tool/openvino/tools/benchmark/utils/statistics_report.py

```diff
@@ -98,7 +152,74 @@ def dump_performance_counters_sorted(self, prof_sorted_info):
             f.write('\n\n')            
         logger.info(f'Sorted performance counters report is stored to {filename}')
 
+class JsonStatisticsReport(StatisticsReport):
+    def __init__(self, config):
+        StatisticsReport.__init__(self, config)
+
+    def dump(self):
+        def list_to_dict(parameters: List[Tuple[str, str]]) -> Dict[str, str]:
+            parameters_as_dict: Dict[str, str] = {}
+            for key, value in parameters:
+                parameters_as_dict[key] = value
+            return parameters_as_dict
+
+        filename = os.path.join(self.config.report_folder, 'benchmark_report.json')
+        with open(filename, 'w') as file:
+            json_statistics = {}
+            if self.Category.COMMAND_LINE_PARAMETERS in self.parameters.keys():
+                json_statistics["cmd_options"] = \
+                    list_to_dict(self.parameters[self.Category.COMMAND_LINE_PARAMETERS])
+
+            if self.Category.RUNTIME_CONFIG in self.parameters.keys():
+                json_statistics["configuration_setup"] = \
+                    list_to_dict(self.parameters[self.Category.COMMAND_LINE_PARAMETERS])
+
+            if self.Category.EXECUTION_RESULTS in self.parameters.keys():
+                json_statistics["execution_results"] = \
+                    list_to_dict(self.parameters[self.Category.COMMAND_LINE_PARAMETERS])
+
+            json.dump(json_statistics, file)
+            logger.info(f"Statistics report is stored to {file.name}")
+
     def dump_performance_counters(self, prof_info_list):
+        def profiling_info_to_dict_list(prof_info_list):
+            total = timedelta()
+            total_cpu = timedelta()
+
+            list: List[Dict[str, str]] = []
+            for profiling_info in prof_info_list:
+                for info in profiling_info:
+                    node_info_dict = {}
+                    node_info_dict['name'] = info.node_name
+                    node_info_dict['node_type'] = info.node_type
+                    node_info_dict['status'] = str(info.status)
+                    node_info_dict['real_time'] = f"{info.real_time / timedelta(milliseconds=1):.3f}"
+                    node_info_dict['cpu_time'] = f"{info.cpu_time / timedelta(milliseconds=1):.3f}"
+                    node_info_dict['exec_type'] = info.exec_type
```

Comment:
```suggestion
                    node_info_dict = {
                        'name': info.node_name,
                        'node_type': info.node_type,
                        # ... rest of variables
                    }
```
It looks cleaner being initialized this way.

Reply:
Agree, applied!